script/lib/l10n/grd_string_replacements.py

```diff
@@ -6,65 +6,113 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 
+def get_branding_replacements(brand=None):
```

Comment:
don't use `=None` in all these scripts. Make it a required variable and let it fail if the value is not passed/invalid.